package.json

```diff
@@ -1,73 +1,71 @@
 {
-  "name": "mpc-phase2-suite",
-  "description": "MPC Phase 2 suite of tools for conducting zkSNARKs trusted setup ceremonies",
-  "repository": "git@github.com:quadratic-funding/mpc-phase2-suite.git",
-  "homepage": "https://github.com/quadratic-funding/mpc-phase2-suite",
-  "bugs": "https://github.com/quadratic-funding/mpc-phase2-suite/issues",
-  "license": "MIT",
-  "private": true,
-  "workspaces": {
-    "packages": [
-      "phase2cli",
-      "firebase"
+    "name": "zkmpc",
+    "description": "MPC Phase 2 suite of tools for conducting zkSNARKs trusted setup ceremonies",
+    "repository": "git@github.com:quadratic-funding/mpc-phase2-suite.git",
+    "homepage": "https://github.com/quadratic-funding/mpc-phase2-suite",
+    "bugs": "https://github.com/quadratic-funding/mpc-phase2-suite/issues",
+    "license": "MIT",
+    "private": true,
+    "keywords": [
+        "typescript",
+        "zero-knowledge",
+        "zk-snarks",
+        "phase-2",
+        "trusted-setup",
+        "ceremony",
+        "snarkjs",
+        "circom"
     ],
-    "nohoist": [
-      "**/snarkjs",
-      "**/snarkjs/**"
+    "scripts": {
+        "build": "lerna run build",
+        "pretest": "yarn build",
+        "test": "export GOOGLE_APPLICATION_CREDENTIALS=\"./packages/backend/serviceAccountKey.json\" && jest --config=jest.json --detectOpenHandles --forceExit  --coverage",
```

Comment:
`yarn test` fails due to missing file(`./packages/backend/serviceAccountKey.json`). Test should pass by default.

Reply:
Also when all .env files are setup, while tests pass, this is the output of jest:

`error Command failed with exit code 1.`

Reply:
The `serviceAccountKey.json` is needed for the execution of Admin SDK (used in the test harness for e2e test). I tried to setup a simple GH action for testing CI, but seems we reached the same problem. You can find more here #208

Reply:
> Also when all .env files are setup, while tests pass correctly, this is the output of jest:
> 
> `error Command failed with exit code 1.`

This happens because we are not reaching the expected coverage threshold. This error should expire automatically when reaching that.

Reply:
> > Also when all .env files are setup, while tests pass correctly, this is the output of jest:
> > `error Command failed with exit code 1.`
> 
> This happens because we are not reaching the expected coverage threshold. This error should expire automatically when reaching that.

Right makes sense

Reply:
> The `serviceAccountKey.json` is needed for the execution of Admin SDK (used in the test harness for e2e test). I tried to setup a simple GH action for testing CI, but seems we reached the same problem. You can find more here #208

But test should pass locally to check changes of code doesn't break codebase before submit PR. (I suggested to have local e2e test in https://github.com/quadratic-funding/mpc-phase2-suite/pull/208#issuecomment-1330472423).

Reply:
> > Also when all .env files are setup, while tests pass correctly, this is the output of jest:
> > `error Command failed with exit code 1.`
> 
> This happens because we are not reaching the expected coverage threshold. This error should expire automatically when reaching that.

Just wanted mention that `coverageThreshold` is for lower limit of coverage.

> This will be used to configure minimum threshold enforcement for coverage results. 
https://jestjs.io/docs/configuration#coveragethreshold-object

Reply:
> > The `serviceAccountKey.json` is needed for the execution of Admin SDK (used in the test harness for e2e test). I tried to setup a simple GH action for testing CI, but seems we reached the same problem. You can find more here #208
> 
> But test should pass locally to check changes of code doesn't break codebase before submit PR. (I suggested to have local e2e test in [#208 (comment)](https://github.com/quadratic-funding/mpc-phase2-suite/pull/208#issuecomment-1330472423)).

Just replied to this on the thread :)

Reply:
> > > > Also when all .env files are setup, while tests pass correctly, this is the output of jest:
> > > > `error Command failed with exit code 1.`
> > > 
> > > 
> > > This happens because we are not reaching the expected coverage threshold. This error should expire automatically when reaching that.
> > 
> > 
> > Just wanted mention that `coverageThreshold` is for lower limit of coverage.
> > > This will be used to configure minimum threshold enforcement for coverage results.
> > > https://jestjs.io/docs/configuration#coveragethreshold-object
> 
> Yes, we have set 90% and 95% as lower limits for jest! So, that's why we are getting the unexpected termination right now!

I figured out what might be a problem: It causes false failure. How could we distinguish it from actual failure, without check the logs for every runs?

Reply:
It makes sense. `yarn coverage` or `yarn test:coverage` or whatever would work.

Reply:
Lovely, thanks!